app/helpers/shows_helper.rb

```diff
@@ -0,0 +1,11 @@
+require 'csv'
+
+module ShowsHelper
+  def get_csv_data
+    data = []
+    CSV.foreach("lib/assets/netflix_titles.csv", headers: true) do |row|
```

Comment:
you could actually make this whole thing a 1 liner by doing this:
`CSV.foreach("lib/assets/netflix_titles.csv", headers: :true).map(&:to_h)`
and also you should put this `lib/assets/netflix_titles.csv` into a constant variable in this file which you can use here.

Reply:
or like this `CSV.foreach("lib/assets/netflix_titles.csv", headers: :true).each_with_object([]) { |obj, array| array << obj.to_h }` if you want a histogram...remember this one, we might need it when we import csv into AR models :D